openmdao/core/vec_wrapper.py

```diff
@@ -943,97 +943,94 @@ def _get_flattened_sizes(self):
         return [[(n, acc.meta['size']) for n, acc in iteritems(self._dat)
                         if not acc.pbo]]
 
-    def distance_along_vector_to_limit(self, alpha, duvec, vector_alpha):
+    def distance_along_vector_to_limit(self, alpha, duvec):
         """ Returns a new alpha so that new_u = current_u + alpha*duvec does
         not violate any `lower` or `upper` limits if specified.
 
         Args
         -----
-        alpha: float or ndarray
+        alpha: ndarray
             Initial value for step in gradient direction.
         duvec: `Vecwrapper`
             Direction to apply step. generally the gradient.
-        vector_alpha: bool
-            If True, then alpha is a vector, so limit each vector element
-            individually.
 
         Returns
         --------
-        float or ndarray
+        ndarray
             New step size(s), backtracked to prevent violation."""
 
         # A single index of the gradient can be zero, so we want to suppress
         # the warnings from numpy.
         old_warn = numpy.geterr()
         numpy.seterr(divide='ignore')
```

Comment:
we should really wrap the code below in a try block and reset the numpy error behavior in a finally block.